pydle/client.py

```diff
@@ -118,7 +117,7 @@ async def connect(self, hostname=None, port=None, reconnect=False, **kwargs):
 
         # Disconnect from current connection.
         if self.connected:
-            await self.disconnect(expected=True)
```

Comment:
Revert this change.
While it is the default, expressly passing the value more clearly conveys intent